src/RichText/cssToReactStyle.ts

```diff
@@ -1,10 +1,12 @@
+import React from 'react';
+
 export function cssToReactStyle(cssString?: string): React.CSSProperties {
 
 	if (!cssString) {
 		return {};
 	}
 
-	const styleObject: React.CSSProperties = {};
+	const styleObject: Record<string, any> = {};
```

Comment:
Why? Is React.CSSProperties too strict? What about Partial<React.CSSProperties>

Personally I would avoid `any` so if you still want just object I would go for Record<string, unknown>